flopy/discretization/grid.py

```diff
@@ -72,8 +94,6 @@ class Grid:
         bottom elevations of all cells
     idomain : int or ndarray
         ibound/idomain value for each cell
-    crs : pyproj.CRS
```

Comment:
It was moved [further down to the property](https://github.com/modflowpy/flopy/pull/1850/files/5874a9a516ab0bf82e5887b33dd9bf68dd7ce1a6#diff-1878877c777302653c7788beb4db8864814491d105a21739d20363411aa292d7R295-R301), which would render [here](https://flopy.readthedocs.io/en/3.4.1/source/flopy.discretization.grid.html#id4) (currently blank). With Sphinx it is cross-referencable using ``:py:attr:`crs``. I might get around to moving the other property docs later, since it's easier to see and xref property docs beside the relevant code.